client/src/dbus/collection.rs

```diff
@@ -133,7 +133,7 @@ impl<'a> Collection<'a> {
                         Arc::clone(&self.session),
                         self.algorithm,
                         item,
-                        self.aes_key.as_ref().map(Arc::clone),
+                        self.aes_key.clone(),
```

Comment:
Please drop this commit, fixed more clippy warnings + those in main just now

Reply:
will do, thanks!